setup.py

```diff
@@ -228,6 +228,14 @@ def run(self):
         print("Unsupported command for openshift-ansible")
 
 
+def extract_requirements(filename):
+    """
+    Extracts requirements from a pip formatted requirements file.
+    """
+    with open(filename, 'r') as requirements_file:
+        return [x.split()[0] for x in requirements_file.readlines()]
```

Comment:
I'm wondering if this function is worth it.
Might work for our specific files now, but doesn't work for arbitrary `requirement.txt` files. They can contain version numbers, `-e ~/path/to/sth`, etc.